js/server/tests/import/import.js

```diff
@@ -441,6 +441,37 @@ function importTestSuite () {
       assertEqual(expected, actual);
     },
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test database creation
+////////////////////////////////////////////////////////////////////////////////
+    testCreateDatabase : function () {
+      var db = require("@arangodb").db;
+      var testdb = db._useDatabase("UnitTestImportCreateDatabase");
```

Comment:
Can we wrap this db._useDatabase() into some try { db._useDatabase(...); ... } finally { db._useDatabase("_system"); }?
Otherwise the test will end up in a different database than started, which may have side effects (and will make following tests fail if some are added).